src/lib/mongoBackend/MongoGlobal.h

```diff
@@ -150,9 +150,15 @@ using namespace mongo;
 *
 * mongoConnect -
 */
-extern bool mongoConnect(const char* host, const char* db, const char* rplSet, const char* username, const char* passwd, bool multitenant);
-extern bool mongoConnect(const char* host);
+extern bool mongoConnect(const char* host,
+                         const char* db,
+                         const char* rplSet,
+                         const char* username,
+                         const char* passwd,
+                         bool        _multitenant,
+                         double      timeout);
 #ifdef UNIT_TEST
+extern bool mongoConnect(const char* host);
```

Comment:
I have realized that the mongoConnect() variant using just the host name is only used in unit test. Thus, is safer including it under this `#ifdef` in order to avoid bypassing the timeout (and other parameters) in the case of production code.

NTC